### DISCUSSION TITLE
Sort grouping in descending order

### DISCUSSION THREAD

**Discussion by AmberKonkolEHL** [microsoft-search/pnp-modern-search#4139] [Q&A] [answered]
Is there a way for me to customize the Details List layout to have the grouping sort in descending order?   

Trying to group the results by a site column - Year - and have the current year at the top. 

Thanks!

**Comment by kasperbolarsen** [comment]
Is Year a site column field based on a choice?

**Reply by AmberKonkolEHL** [reply]
It is a managed meta data/term set I linked to a refinable string.

**Comment by kasperbolarsen** [comment, marked as answer]
Currently it is not possible to sort the group by property.  I will tag your request as a feature request.
Will you or someone in your organization be able to implement this Pull Request?

**Reply by AmberKonkolEHL** [reply]
I have never done a pull request before, but I am happy to provide assistance however I can.  I am fairly green around the gills when it comes to interacting with GitHub.
